client/src/main/java/io/confluent/kafka/schemaregistry/client/SchemaRegistryClientConfig.java

```diff
@@ -58,4 +65,16 @@ private static ConfigDef.Importance importanceFor(
     return ConfigDef.Importance.valueOf(importance.name());
   }
 
+  public static Integer getSchemaRegistryHttpConnectTimeoutMs(Map<String, ?> configs) {
+    return configs != null && configs.containsKey(SCHEMA_REGISTRY_HTTP_CONNECT_TIMEOUT_MS)
+        ? (Integer) configs.get(SCHEMA_REGISTRY_HTTP_CONNECT_TIMEOUT_MS)
```

Comment:
Probably should do `Integer.parseInt(configs.get(HTTP_CONNECT_TIMEOUT_MS).toString())` instead since when passed through a properties file, it could be a string.

Is it possible to add a test of passing the property via a properties file?